modules/programs/librewolf.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.librewolf;
+
+  mkOverrides = prefs: ''
+    // Generated by Home Manager.
+
+    ${concatStrings (mapAttrsToList (name: value: ''
+      defaultPref("${name}", ${builtins.toJSON value});
+    '') prefs)}
+  '';
+
+in {
+  meta.maintainers = [ maintainers.onny ];
+
+  options.programs.librewolf = {
+
+    enable = mkEnableOption "Librewolf browser, privacy enhanced Firefox fork";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.librewolf;
+      defaultText = literalExpression "pkgs.librewolf";
+      description = "The LibeWolf package to use.";
+    };
+
+    overrides = mkOption {
```

Comment:
I would suggest calling this option `settings`, which is more in line with the naming inside other modules. The overrides name is used more for overrides in the Nix sense.